Automation/utils/types.ts

```diff
@@ -0,0 +1,89 @@
+export interface Args {
+  RN_VERSION: string;
+  ITERATIONS: number;
+}
+
+export interface Config {
+  BASE_DIR: string;
+  TEMPLATE_DIR: string;
+  ERROR_LOG_FILE: string;
+}
+
+export type LogLevel = 'ERROR' | 'INFO';
+
+export interface BaseState {
+  version: string;
+  projectCreated: boolean;
+  templatesApplied: boolean;
+  reportsGenerated: boolean;
+  startTime: string;
+  packageName?: string;
+}
+
+export interface AndroidState {
+  androidOldArchTested: boolean;
+  androidNewArchTested: boolean;
+  androidOldArchApkPath?: string;
+  androidNewArchApkPath?: string;
+  androidOldArchMetadataPath?: string;
+  androidNewArchMetadataPath?: string;
+}
```

Comment:
```suggestion
export interface AndroidState {
  androidOldArchTested: boolean;
  androidNewArchTested: boolean;
  androidOldArchApkPath?: string;
  androidNewArchApkPath?: string;
  androidOldArchMetadataPath?: string;
  androidNewArchMetadataPath?: string;
}
```

Does Android state need Android prefix for all the properties?